conf/machine/ray.conf

```diff
@@ -0,0 +1,18 @@
+
+#@TYPE: Machine
+#@NAME: Huawei Watch
+#@DESCRIPTION: Machine configuration for the Huawei Watch
```

Comment:
You might want to change the values here ;)

Reply:
Great catch ;P